frontend/src/components/maps/BuildingPopupView.scss

```diff
@@ -5,25 +5,32 @@
   max-width: 250px;
   box-shadow: 1px 2px 2px;
   text-align: left;
-  padding: 12px;
+  padding: 10px;
 
   .row .col :first-child {
     margin-top: 0px;
   }
 
   .label {
     font-weight: 700;
+    padding-right: 5px;
+  }
+  .name {
+    font-weight: bold;
   }
 
   .list-group {
     margin-top: 6px;
   }
   .list-group-item {
-    padding: 0;
+    padding: 0px 0px 5px 0px !important;
```

Comment:
ugh, another `!important`... styling is a mess

Reply:
I don't think this is necessary. Map.scss is causing this class not to work. That was a change that Goeme introduced for the layers popup. Ideally the styling would be consistent, if not we need a specific class for that popup so that we can differentiate it.

Reply:
Just as a note, in your browser you can see what css is overriding your css, which should help you determine if the override is from conflicting _style_ _bootstrap_ or our css. Apologies if you were already aware of this, but we should avoid falling back on important where possible, such as instances like this.

Reply:
![image](https://user-images.githubusercontent.com/3180256/98276659-26dda400-1f4b-11eb-8750-7c4ecb8c1eaa.png)
This is the cause of the issue and why I needed to use `!important`.  How do I fix this?

Reply:
We can fix by doing this: `
.parcelPopup {
  .row {
    .col {
      .list-group {
        margin-top: 6px;
        .list-group-item {
         padding: 0px 0px 5px 0px
        }
      }
    }
  }
}`

Reply:
Or for a building: 
`.buildingPopup { .row { .col { .list-group { margin-top: 6px; .list-group-item { padding: 0px 0px 5px 0px } } } } }`